mage_ai/tests/data_preparation/models/test_block.py

```diff
@@ -29,6 +32,88 @@ def test_create(self):
         self.assertEqual(block2.uuid, 'test_data_loader')
         self.assertEqual(block2.type, 'data_loader')
 
+    def test_execute(self):
+        pipeline = Pipeline.create('test pipeline', self.repo_path)
+        block1 = Block.create('test_data_loader', 'data_loader', self.repo_path, pipeline)
+        block2 = Block.create('test_transformer', 'transformer', self.repo_path, pipeline)
+        block2.upstream_blocks = [block1]
+        block1.downstream_blocks = [block2]
+        with open(block1.file_path, 'w') as file:
+            file.write('''import pandas as pd
+@data_loader
+def load_data():
+    data = {'col1': [1, 1, 3], 'col2': [2, 2, 4]}
+    df = pd.DataFrame(data)
+    return [df]
+            ''')
+        with open(block2.file_path, 'w') as file:
+            file.write('''import pandas as pd
+@transformer
+def remove_duplicate_rows(df):
+	df_transformed = df.drop_duplicates()
```

Comment:
nit: use 4 spaces indentation

Reply:
i'm not sure why it looks like this in the PR, but it only shows a 4 space indent in my editor